waspc/waspls/src/Wasp/LSP/Commands/ScaffoldTsSymbol.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Wasp.LSP.Commands.ScaffoldTsSymbol
```

Comment:
I think I expected that it would scaffold code besides just functions, but now after implementing it I don't see that happening. So let's rename it! How about `ScaffoldTsFunction` (and renaming the command to `wasp.scaffold.ts-function`). Or maybe we should do something more specific to external imports in the name?

Reply:
Aha I see now below that we are actually tailored for ExtImports here, ok in that case it might make sense to make that clear in the name.

Reply:
I think that might not be necessary. It seems unlikely that waspls will add more TS scaffolding besides just for external imports, so maybe not worth specifying that in the name.

Reply:
Ok, whatever you think is best here then!